example/test_problem/ELBDM/LSS_Hybrid_Zoomin/plot_script/plot_projection_grid.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+import sys
+import yt
+
+# global parameter settings
+field         = ('gas', 'density')
+axes          = ["x", "y", "z"]                            # directions
+lv            = 10                                         # maximum level for sampling AMR grid
+dpi           = 300
+colormap_dens = 'algae'
+center_mode   = 'c'
+zooms         = [1, 7]
+# load the command-line parameters
+parser = argparse.ArgumentParser( description='Projection of mass density' )
+
+parser.add_argument( '-i', action='store', required=False, type=str, dest='prefix',
+                     help='path prefix [%(default)s]', default='../' )
+parser.add_argument( '-s', action='store', required=True,  type=int, dest='idx_start',
+                     help='first data index' )
+parser.add_argument( '-e', action='store', required=True,  type=int, dest='idx_end',
+                     help='last data index' )
+parser.add_argument( '-d', action='store', required=False, type=int, dest='didx',
+                     help='delta data index [%(default)d]', default=1 )
+
+args=parser.parse_args()
+
+# take note
+print( '\nCommand-line arguments:' )
+print( '-------------------------------------------------------------------' )
+for t in range( len(sys.argv) ):
+   print(str(sys.argv[t]))
+print( '' )
+print( '-------------------------------------------------------------------\n' )
+
+
+idx_start = args.idx_start
+idx_end   = args.idx_end
+didx      = args.didx
+prefix    = args.prefix
+
+center = [0.295,9.522,8.27] # center of the targeted halo in the zoom-in simulation
```

Comment:
`Input__ZoombinBox` contains the approximate center for the host halo only, but not the subhalo. 
Leaving it like that is more general,  instead of reading `Input__ZoombinBox` ,